pulp_rpm/app/migrations/0009_revision_null.py

```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='rpmrepository',
             name='last_sync_revision_number',
-            field=models.CharField(max_length=20, null=True),
+            field=models.CharField(max_length=20),
```

Comment:
It seems to work without re-modifying 0005, I think because ```null=False``` by default.  Should I modify 0005 anyway?

Reply:
Just removing it is fine I believe.

Reply:
Oh, I'm not sure, but I'd remove from 0005 as well. Less flipping it back and forth, and 0005 will be in it's original state.